README.md

```diff
@@ -308,8 +308,70 @@ just added.
 ## API
 
 <generated_api_start />
+#### The Jest global environment
+
+In your test files, Jest puts each of these methods and objects into the global environment. You don't have to require or import anything to use them.
+
+  - `afterEach(fn)`
+  - `beforeEach(fn)`
+  - [`check`](https://facebook.github.io/jest/docs/api.html#check)
```

Comment:
actually check and gen are not global. They can be required through:

```js
const {check, gen} = require('jest-check');
```

I think it is fine for now not to document this. I am even thinking of getting rid of this from Jest again because no one uses it.